getting_started/brainvision.md

```diff
@@ -50,24 +50,26 @@ For validation of BrainVision file triplets, you can use the [brainvision-valida
 
 ## Preprocessing of raw EEG data
 
-FieldTrip needs the user to define what file to read in. The BrainVision Recorder software usually stores different filetypes (.vhdr, .eeg, .vmrk). For reading the data into FieldTrip you can refer to the .eeg file, for exampl
+FieldTrip needs the user to define what file to read in. The BrainVision Recorder software usually stores different filetypes (.vhdr, .dat, .vmrk). For reading the data into FieldTrip you can refer to the .dat file, for example. In order for FieldTrip to read the data file, it has to be exported as a .dat file from BrainVision.
 
     cfg = [];
-    cfg.dataset = '/users/karlheinz/EEG/myrecording.eeg';
+    cfg.dataset = '/users/karlheinz/EEG/myrecording.dat';
     ...
 
-The .eeg files are the raw data files, i.e. they contain the data as it has been stored upon acquisition.
+The .dat files are the raw data files, i.e. they contain the data as it has been stored upon acquisition.
 
 You can subsequently epoch your data using [ft_definetrial](https://github.com/fieldtrip/fieldtrip/blob/release/ft_definetrial), and you can read in the data and preprocess it using [ft_preprocessing](https://github.com/fieldtrip/fieldtrip/blob/release/ft_preprocessing). Note that in FieldTrip, no unit conversion takes place.
 
+Even for raw data, the data are considered segmented by FieldTrip. The data file is considered as a very long segment starting with the event "New Segment" and ending at the end of the recording. As a way to investigate the specificities of the BrainVision file, I would recommend always checking the .vmrk file by either opening it in a text editor, or using the function ft_definetrial with configuration cfg.trialdef.eventtype = '?'.
+
 ## Exporting raw EEG data after doing processing in BrainVision Analyzer
 
 Sometimes users have already done some processing (e.g., rereferencing, epoching, artifact identification) in BrainVision Analyzer, and in order to avoid repeating the time consuming / subjective selection steps, it might be preferable to start from the processed data. BrainVision Analyzer stores the processing steps in a so called history file, keeping the raw data unchanged, and applying the processing steps on-the-fly. This is not something that FieldTrip can work with, so you need to export your data first.
 
 The following describes the recipe to export the processed data into a format the FieldTrip can deal with. It produces a triplet of files (.vhdr, .vmrk and .seg (instead of .eeg)), that can be imported into FieldTrip, in much the same way as described above.
```

Comment:
shouldn't 'seg' be replaced by 'dat' here as well?

Reply:
Oh, my bad, I missed this one!